internal/output/default.go

```diff
@@ -5,29 +5,24 @@
 package output
 
 import (
-	"errors"
 	"fmt"
 	"os"
-	"reflect"
+
+	"github.com/sirupsen/logrus"
 )
 
 // Default outputs the provided input to stdout.
 func Default(_input interface{}) error {
-	// check if the input provided is nil
-	if _input == nil {
-		return errors.New("empty value provided for default output")
-	}
+	logrus.Debugf("creating output with %s driver", DriverDefault)
 
-	// check if the value of input provided is nil
-	//
-	// We are using reflect here due to the nature
-	// of how interfaces work in Go. It is possible
-	// for _input to be a non-nil interface but the
-	// underlying value to be empty or nil.
-	if reflect.ValueOf(_input).IsZero() {
-		return errors.New("empty value provided for default output")
+	// validate the input provided
+	err := validate(DriverDefault, _input)
```

Comment:
Each function should call `validate()` to help ensure the input meets our standards before outputting to `stdout`